README.md

```
@@ -1,23 +1,109 @@
-## API
-
-
 ## Usage
 ```js
 const SqsEmitter = require('@janiscommerce/sqs-emitter');
+```
+
+### SQS Emitter
+
+> This class is compatible with [@janiscommerce/api-session](https://npmjs.com/@janiscommerce/api-session). If it's instanciated using `getSessionInstance`, a message attribute `janis-client` with the session's `clientCode` will be automatically added to every event.
 
```

markdownlint-cli2 (0.17.2)

Blank line inside blockquote
null

(MD028, no-blanks-blockquote)